connectors/cli.py

```diff
@@ -132,7 +132,7 @@ def run(args):
     if args.action == "list":
         logger.info("Registered connectors:")
         for source in get_data_sources(config):
-            logger.info(f"- {source.__doc__.strip()}")
```

Comment:
`__doc__` is different from `name`

it takes the class docstring. 

```
class X:
  """This is the docstring
  """
```

so if we drop this, no need to strip. raises the question about wether we want to keep a description or not